pkg/controller/core/core.go

```diff
@@ -28,11 +28,26 @@ import (
 	"sigs.k8s.io/kueue/pkg/queue"
 )
 
-const updateChBuffer = 10
+const (
+	updateChBuffer                   = 10
+	defaultRequeuingBaseDelaySeconds = 10
+)
+
+type ControllerOptions struct {
+	requeuingBaseDelaySeconds int32
+}
+
+type ControllerOption func(*ControllerOptions)
+
+func WithControllerRequeuingBaseDelaySeconds(value int32) ControllerOption {
```

Comment:
This seems unnecessary for the cherry-pick

Reply:
I use it to pass different base in prod and integration tests. When I use 10s in integration tests they fail, as the Timeout is 5s only. I considered the following approaches:
1. Expose the configuration via API (deferred to a follow up since this shouldn't be cherry-picked)
2. Bump the timeout for the subset of integration tests for PodsReady - this would work, but seems wasteful
3. Expose configuration which allows me to pass different values to `SetupControllers` (chosen)

Let me know if there is another approach.

Reply:
oh ok, let's keep this approach.